### ISSUE TITLE
Support NVDA >= 2018.3 (wxPython 4)

### ISSUE THREAD

**Issue by JulienCochuyt** [derekriemer/nvda-notepadPlusPlus#15]
From 2018.3 on, NVDA migrated from wxPython 3 (Classic) to 4 (Phoenix).
This was a big preliminary step towards migrating to Python 3.

In wxPython 4, a few constructs have changed.
Especially, the symbol `wx.PyDeadObjectError` has been abandoned, a regular `RuntimeError` is now raised instead.

Reference: [Classic vs Phoenix](https://wxpython.org/Phoenix/docs/html/classic_vs_phoenix.html)

This add-on references `wx.PyDeadObjectError` in `addon/appModules/notepad++/addonGui.py`
line 45 in method `GuiManager.__del__`
This causes an `AttributeError` to be raised during shutdown of NVDA, which can lead on some systems to a failure of the shutdown process with the watchdog process restarting NVDA infinitely. The user is then unable to exit NVDA nor launch another version and forced to restart the computer to do so.

**Comment by derekriemer** [comment]
This is now fixed.
